packages/@lwc/ssr-compiler/src/compile-js/generate-markup.ts

```diff
@@ -106,6 +117,8 @@ export function addGenerateMarkupExport(
                 mutationTracker: '__mutationTracker',
                 renderAttrs: '__renderAttrs',
                 SYMBOL__SET_INTERNALS: '__SYMBOL__SET_INTERNALS',
+                establishContextfulRelationship: '__establishContextfulRelationship',
+                connectContext: '__connectContext',
```

Comment:
TODO: add this through `cxt.hoist` in the if block guarded by `state.wireAdapters.length`

Reply:
Strike that, we can't do `cxt.hoist` - that is only available in template compilation.